### PR TITLE
#12 Commit message validation

### PR DESCRIPTION
**Pull request by sarawallebo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,3 +42,26 @@ You don’t have to ever use `eject`. The curated feature set is suitable for sm
 You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
 
 To learn React, check out the [React documentation](https://reactjs.org/).
+
+## Commit Message Convention
+
+This project enforces Conventional commits. See [Conventional Commits specification](https://www.conventionalcommits.org/).
+
+In short:
```
Comment (Evanion) [inline README.md]: you could add an example of how a commit message would be.
`docs[tooling]: added documentation on commit msg validation`

**Review by Evanion — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (saraervik) [reply README.md]: fixed!

**Comment by saraervik** [comment]
commitlint does not seem to run on the new setup with meta project... need to fix before merging

**Comment by saraervik** [comment]
solved the problem, yarn needs to be run to download packages (husky etc).

**Pull request merged**